Assets/HoloToolkit/Utilities/Scripts/CameraCache.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public static class CameraCache
+    {
+        private static Camera cachedCamera;
+
+        /// <summary>
+        /// Returns a cached reference to the main camera and uses Camera.main if it hasn't been cached yet.
+        /// </summary>
+        public static Camera main
```

Comment:
Although this is very optional because it's following the same pattern as unity to get the main camera. Less work to refactor, but maybe making it PascalCase will show there's a difference between usages.